docs/01-writing-tests.md

```diff
@@ -267,14 +267,18 @@ test('context is unicorn', t => {
 });
 ```
 
-## Retrieving test meta data
+## Retrieving test metadata
 
-Helper files can determine the filename of the test being run by reading `test.meta.file`.  This eliminates the need to pass `__filename` from the test to helpers.
+Access metadata of the test being run by reading `test.meta`.
```

Comment:
I try to keep this short. Found the previous text a little redundant.

Reply:
Yea… helper files are less emphasized within AVA core anyhow.